### DISCUSSION TITLE
Service Not started automatically

### DISCUSSION THREAD

**Discussion by peterlymo** [Security-Onion-Solutions/securityonion#8566] [Unsupported Versions] [answered]
I deployed a security onion standalone and was working well, and even if its rebooted all services started automatically with no delay, 

But currently services not started automatically its freeze on WAIT_START or STARTING only so-aptcacheorg, so-dockerregistry and so-kratos only stated well 

I tried 
So-start all(not working,took forever to finish)

Though i tried so-wazuh-start manually worked

**Comment by dougburks** [comment, marked as answer]
It sounds like your system was working properly and then something changed and it stopped working properly. 

Did you make any changes to the system?

**Reply by peterlymo** [reply]
I didn't, late reply, i had to reinstall SO , although i have another error, so-elastalert seems to be missing, when I restart it it say no such a container

**Reply by dougburks** [reply]
Please start a new discussion with an appropriate title for your new error. Marking this discussion as answered.
